connectors/ssh/src/main/java/org/crsh/ssh/term/URLKeyPairProvider.java

```diff
@@ -53,7 +54,7 @@ public URLKeyPairProvider(Resource key) {
     List<KeyPair> keys = new ArrayList<KeyPair>();
     if (key != null) {
       try {
-          Object o = KeyPairUtils.readKey(new InputStreamReader(new ByteArrayInputStream(key.getContent())));
+          Object o = KeyPairUtils.readKey(new InputStreamReader(new ByteArrayInputStream(key.getContent()), Charset.forName("UTF-8")));
```

Comment:
It would be better if this was settable, rather than forcing an encoding, especially if the default charset is not desired.